### PR TITLE
feat(website): revamp home page UI

### PR DESCRIPTION
**Pull request by lamvh**
image

image

### CODE REVIEW

**Comment by lamvh** [comment]
@duongdev @bkdev98  please help me review.

File: src/app/layout.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 import type { Metadata } from "next";
-import { Be_Vietnam_Pro } from "next/font/google";
+import { Inter } from "next/font/google";
 import { PHProvider } from "@/components/PHProvider";
 import { Header } from "@/components/header";
 
-const beVietnamPro = Be_Vietnam_Pro({
-  subsets: ["latin"],
-  weight: ["300", "400", "500", "600"],
+const beVietnamPro = Inter({
```
Comment (duongdev) [inline src/app/layout.tsx]: Also change variable name

File: src/components/header.tsx [hunk 2]
```diff
@@ -1,34 +1,28 @@
...
-  )
-}
\ No newline at end of file
+	return (
+		<div>
+			<div className="fixed pointer-events-none z-40 inset-0 h-16 bg-[hsla(0,0%,4%,.2)] backdrop-blur-md [maskImage:linear-gradient(to_bottom,#000_20%,transparent_calc(100%-20%))]" />
+			<div className="fixed border-b w-full border-muted-foreground h-16 top-0 backdrop-blur-xl" />
+			<header className="fixed flex max-w-4xl mx-auto overflow-hidden z-50 h-12 saturate-150 top-2 left-4 right-4">
+				<div className="flex mx-auto justify-between items-center w-full">
+					<a href="/" className="flex items-center gap-2">
```
Comment (duongdev) [inline src/components/header.tsx]: Use Link

File: src/app/page.tsx [hunk 3]
```diff
@@ -1,109 +1,186 @@
...
+					<section className="flex h-full flex-col pt-20 md:w-2/3 w-full">
+						<div className="flex flex-col md:items-start items-center space-y-6 md:space-y-5">
+							<h1 className="md:text-6xl text-5xl font-medium text-foreground">
+								Manage your expenses & budgets seamlessly.
+							</h1>
+							<p className="text-muted-foreground">
+								We've just launched! Get your AI finance assistant for free.
+							</p>
+							<a
+								href="https://apps.apple.com/vn/app/6pm-finance-intelligence/id6504689878"
```
Comment (duongdev) [inline src/app/page.tsx]: Use constant

File: src/components/header.tsx [hunk 4]
```diff
@@ -1,34 +1,28 @@
...
+						<img src="/logo.svg" alt="6pm" className="h-9 w-9" />
+						<div className="text-foreground text-xl font-medium">6pm</div>
+					</a>
+					<nav className="flex items-center gap-2">
+						<a
+							href="https://apps.apple.com/vn/app/6pm-finance-intelligence/id6504689878"
```
Comment (duongdev) [inline src/components/header.tsx]: Use constant

**Review by duongdev — CHANGES_REQUESTED** [review]

File: src/components/header.tsx [hunk 5]
```diff
@@ -1,34 +1,28 @@
...
-          >
-            <XIcon className="h-4 w-4 fill-white" />
-          </a>
-          <a
-            href="https://github.com/sixpm-ai/6pm"
-            target="_blank"
-            rel="noopener noreferrer"
-            className="inline-flex h-9 w-9 items-center justify-center"
-          >
-            <GithubIcon className="h-4 w-4 fill-white" />
```
Comment (duongdev) [inline src/components/header.tsx:28]: Use constants for links so we can update they easily later

**Review by bkdev98 — APPROVED** [review]

File: src/components/header.tsx [hunk 6]
```diff
@@ -1,34 +1,27 @@
...
-  )
-}
\ No newline at end of file
+	return (
+		<div>
+			<div className="fixed pointer-events-none z-40 inset-0 h-16 bg-[hsla(0,0%,4%,.2)] backdrop-blur-md [maskImage:linear-gradient(to_bottom,#000_20%,transparent_calc(100%-20%))]" />
+			<div className="fixed border-b w-full border-[#A8AAB2]/[.25] h-16 top-0 backdrop-blur-xl" />
+			<header className="fixed flex max-w-4xl mx-auto overflow-hidden z-50 h-12 saturate-150 top-2 left-4 right-4">
+				<div className="flex mx-auto justify-between items-center w-full">
+					<Link href="/" className="flex items-center gap-2">
```
Comment (duongdev) [inline src/components/header.tsx]: gap-3

**Review by duongdev — CHANGES_REQUESTED** [review]
I also left some comments on Vercel preview. Please check.

**Review by duongdev — CHANGES_REQUESTED** [review]
Added comments on Vercel

**Review by duongdev — COMMENTED** [review]
The border color should be configured on tailwind level.

File: tailwind.config.ts [hunk 7]
```diff
@@ -30,6 +31,8 @@ const config: Config = {
         muted: {
           DEFAULT: "hsl(var(--muted))",
           foreground: "hsl(var(--muted-foreground))",
+          background: "hsl(var(--muted-background))",
+          border: "hsl(var(--muted-foreground),0.25)",
```
Comment (duongdev) [inline tailwind.config.ts]:
```suggestion
          border: "hsl(var(--muted-foreground), 0.25)",
```

**Review by duongdev — CHANGES_REQUESTED** [review]

**Review by duongdev — APPROVED** [review]

**Pull request merged**